CollAction/package.json

```diff
@@ -7,31 +7,51 @@
     "build": "webpack",
     "watch": "webpack -w"
   },
-  "author": "Stewart Matheson",
+  "author": "Collaction Team",
   "license": "ISC",
   "devDependencies": {
-    "@types/react": "^15.0.6",
-    "@types/react-dom": "^0.14.22",
+    "@fortawesome/fontawesome-free": "^5.3.1",
```

Comment:
Looks like we are using both CDN and include them in the javascript bundle... I think we should pick one and go with that. I prefer the CDN-solution.. can you check with @logophobia ? Not really sure why a include the fonts stuff in our own codebase, but there must have been a reason for that....

Reply:
font library has been removed, we now use v5 fontawesome which allows for using optimized version of fonts with inline svg generated for each import, it does not require entire font to be downloaded to client browser. It only requires javascript part to be bundled  then react components can use optimized version without downloading from CDN entire fonts of v4. It is small performance improvments for pages that are high traffic ( main page, footer etc. )